pkg/services/object/put/distributed.go

```diff
@@ -177,16 +177,27 @@ func (t *distributedTarget) saveObject(obj objectSDK.Object, objMeta object.Cont
 	// TODO: handle rules in parallel. https://github.com/nspcc-dev/neofs-node/issues/3503
 
 	repRules := t.containerNodes.PrimaryCounts()
+	ecRules := t.containerNodes.ECRules()
 	if typ := obj.Type(); len(repRules) > 0 || typ == objectSDK.TypeTombstone || typ == objectSDK.TypeLock {
 		broadcast := typ == objectSDK.TypeTombstone || typ == objectSDK.TypeLink || (!t.localOnly && typ == objectSDK.TypeLock) || len(obj.Children()) > 0
+
+		useRepRules := repRules
+		if broadcast && len(ecRules) > 0 {
+			useRepRules = make([]uint, len(repRules)+len(ecRules))
+			copy(useRepRules, repRules)
+			for i := range ecRules {
+				useRepRules[len(repRules)+i] = 1
```

Comment:
Technically it's even more important to push ts/lock to all nodes in EC group, how about data+parity here instead of 1?

Reply:
pretty reasonable. Done